test/test_cli.rb

```diff
@@ -1,441 +1,357 @@
 # frozen_string_literal: true
+
 require_relative 'helper'
 require 'sidekiq/cli'
+require 'sidekiq/launcher'
 require 'tempfile'
 
-class Sidekiq::CLI
-  def die(code)
-    @code = code
-  end
-
-  def valid?
-    !@code
-  end
-end
-
-class TestCli < Sidekiq::Test
-  describe 'CLI#parse' do
-
+class TestCLI < Sidekiq::Test
+  describe '#parse' do
     before do
       @cli = Sidekiq::CLI.new
       @opts = Sidekiq.options.dup
+      @logger = Sidekiq.logger
```

Comment:
I'm still thinking how to reset global state for each test more efficiently.
This is the first time I work with `minitest` and I still have to find hooks similar to `rspec`.